### PR TITLE
K8SPSMDB-277 Add Licenses from packages

### PR DESCRIPTION
**Pull request by cap1984**
[K8SPSMDB-277](https://jira.percona.com/browse/K8SPSMDB-277)

GCP requests a complete set of license files from libs used by
the operator for the marketplace certification.

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| one-pod | passed |
| upgrade | passed |
| init-deploy | passed |
| self-healing | passed |
| monitoring | passed |
| limits | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scaling | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| security-context | passed |
| arbiter | passed |
| scheduled-backup | failed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| demand-backup-sharded | passed |
| version-service | passed |
| users | passed |

File: build/Dockerfile [hunk 1]
```diff
@@ -17,6 +17,12 @@ RUN mkdir -p build/_output/bin \
                cmd/manager/main.go \
     && cp -r build/_output/bin/percona-server-mongodb-operator /usr/local/bin/percona-server-mongodb-operator
 
+# Looking for all possible License/Notice files and copying them to the image
+RUN find vendor -regextype posix-extended -iregex '.*(license|notice)(\.md|\.txt|$)' \
+         -exec \
+            sh -c 'mkdir -pv /licenses/$(echo "$0" | sed -E "s/\/(license|notice).*$//gi") \
+                   && cp -v "$0" /licenses/$(echo "$0" | sed -E "s/\/(license|notice).*$//gi")' {} \;
+
```
Comment (delgod) [inline build/Dockerfile:25]: another idea, only explicit depencies
```
awk '{print$1}' go.mod \
    | egrep -v -e '^(replace|require|go|module|\))$' \
    | grep -v "^$" \
    | xargs -I{} sh -c "test -d vendor/{} && ls vendor/{}/LICENSE"
```
decision up to @spron-in

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | failed |
| upgrade | failed |
| upgrade-consistency | passed |
| monitoring | failed |
| self-healing | passed |
| demand-backup | passed |
| limits | passed |
| scaling | passed |
| scheduled-backup | passed |
| monitoring-2-0 | failed |
| operator-self-healing | passed |
| security-context | passed |
| arbiter | passed |
| demand-backup-sharded | failed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |

**Comment by JNKPercona** [comment]
PSMDB operator docker - perconalab/percona-server-mongodb-operator:PR-474-fe04f6c6

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| one-pod | passed |
| upgrade | passed |
| monitoring | passed |
| limits | passed |
| upgrade-consistency | passed |
| scaling | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| security-context | passed |
| self-healing | passed |
| arbiter | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| operator-self-healing | passed |
| demand-backup-sharded | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |

**Review by spron-in — DISMISSED** [review]
lgtm

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| one-pod | passed |
| init-deploy | passed |
| upgrade | passed |
| self-healing | passed |
| monitoring | passed |
| limits | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scaling | passed |
| monitoring-2-0 | passed |
| security-context | passed |
| operator-self-healing | passed |
| scheduled-backup | passed |
| arbiter | passed |
| demand-backup-sharded | failed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |

**Comment by JNKPercona** [comment]
PSMDB operator docker - perconalab/percona-server-mongodb-operator:PR-474-92552afa

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| upgrade | passed |
| self-healing | passed |
| monitoring | passed |
| limits | passed |
| upgrade-consistency | passed |
| scaling | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| security-context | passed |
| arbiter | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| demand-backup-sharded | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |

**Comment by JNKPercona** [comment]
PSMDB operator docker - perconalab/percona-server-mongodb-operator:PR-474-284e96a2

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| upgrade | passed |
| init-deploy | passed |
| monitoring | passed |
| self-healing | passed |
| upgrade-consistency | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| operator-self-healing | passed |
| arbiter | passed |
| security-context | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| demand-backup-sharded | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |

**Review by hors — DISMISSED** [review]

**Pull request merged**